packages/mobile/metro.config.js

```diff
@@ -16,6 +17,7 @@ const getClientAliases = () => {
     'assets',
     'audio',
     'common',
+    'harmony',
```

Comment:
I don't think you need this unless we are doing `import { ... } from 'harmony/xyz'` in the client

Reply:
good point